app/Http/Controllers/Api/V1/SearchController.php

```diff
@@ -236,6 +237,8 @@ public function datasets(Request $request): JsonResponse|BinaryFileResponse
                 $datasetsArray[$i]['dataProviderColl'] = $this->getDataProviderColl($model);
                 $datasetsArray[$i]['team']['id'] = $model['team']['id'];
                 $datasetsArray[$i]['team']['is_question_bank'] = $model['team']['is_question_bank'];
+                $teamDARTemplates = DataAccessTemplate::where([['team_id', $model['team']['id']], ['published', 1]])->get();
```

Comment:
`::where(['team_id' => $model['team']['id'],  'published' => 1 ])`

is more typical? Though the internet is telling me your syntax does work?